dev_guide/builds.adoc

```diff
@@ -5,19 +5,40 @@
 :icons:
 :experimental:
 :toc: macro
+:toclevels: 3
 :toc-title:
 :prewrap!:
 
 toc::[]
 
 == Overview
-A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build] is a process of creating
-runnable images to be used on OpenShift. There are three build strategies:
+
+A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build]
+is a process of creating runnable images to be used on OpenShift.
```

Comment:
s/a process/the process/

Can a (singular) build produce (multiple) images?
If so, suggest "...creating one or more runnable images...".
If not, suggest "..creating a runnable image...".